src/nvim/ops.c

```diff
@@ -3644,6 +3644,10 @@ int do_join(size_t count,
    */
   for (t = (linenr_T)count - 1;; t--) {
     cend -= currsize;
+	if(cend == NULL || curr == NULL){
```

Comment:
Ok. So just to clarify, those rebase commands that execute the unit tests are run as a branch is rebased or merged back into master automatically?